addons/trenches/XEH_PREP.hpp

```diff
@@ -1,16 +1,36 @@
-
+PREP(addDigger);
+PREP(applyCamouflageAttribute);
+PREP(automaticFilePath);
 PREP(canContinueDiggingTrench);
 PREP(canDigTrench);
+PREP(canHelpDiggingTrench);
+PREP(canPlaceCamouflage);
+PREP(canRemoveCamouflage);
 PREP(canRemoveTrench);
 PREP(continueDiggingTrench);
+PREP(deleteCamouflage);
 PREP(handleInteractMenuOpened);
 PREP(handleKilled);
 PREP(handlePlayerChanged);
 PREP(handlePlayerInventoryChanged);
-PREP(handleScrollWheel);
 PREP(handleUnconscious);
+PREP(initCurator);
+PREP(initTrench);
+PREP(initTrench3DEN);
+PREP(placeCamouflage);
 PREP(placeCancel);
 PREP(placeConfirm);
 PREP(placeTrench);
+PREP(progressBar);
+PREP(removeCamouflage);
 PREP(removeTrench);
-PREP(setTrenchPlacement);
+
+if ("surfaceTexture" in (uiNamespace getVariable ["Intercept_cba_capabilities",[]])) then {
```

Comment:
We aren't using Intercept?

Reply:
As far as I am aware ace does not incluede Intercept.

Reply:
Intercept is optionally used here. With intercept this is automatically compatible with every terrain instead of having to add it manually.